addons/l10n_it_edi/models/ir_mail_server.py

```diff
@@ -357,3 +357,17 @@ def _return_error_xml(self, tree):
             if descrizione:
                 output_str += "<li>Errore %s: %s</li>" % (element[0].text, descrizione)
         return output_str + "</ul>"
+
+class IrMailServer(models.Model):
+    _name = "ir.mail_server"
+    _inherit = "ir.mail_server"
+
+    def build_email(self, email_from, email_to, subject, body, email_cc=None, email_bcc=None, reply_to=False,
+                attachments=None, message_id=None, references=None, object_id=False, subtype='plain', headers=None,
+                body_alternative=None, subtype_alternative='plain'):
+
+        if self.env.context.get('wo_return_path') and headers:
+            headers.pop('Return-Path', False)
+        return super(IrMailServer, self).build_email(email_from, email_to, subject, body, email_cc, email_bcc, reply_to,
```

Comment:
Any particular reason for converting keywargs to positional? It may save a few characters but it makes the call stack more brittle ;-)

Reply:
Indeed, I missed that. Thanks